### ISSUE TITLE
Template syntax error ,static files is not a registered tag library..How to resolve this

### ISSUE THREAD

**Issue by nadarajan-s-18673** [CleverProgrammer/codedaddies_list#7]

**Comment by mujahid018** [comment]
I am also facing this problem. I also need the solution.

**Comment by ThomasFurtado** [comment]
Have someone figured out how to solve this??

**Comment by nadarajan-s-18673** [comment]
No, I couldn't able to resolve that error . So can you please give any tips
to resolve that . That would be great

Thank you

**Comment by ThomasFurtado** [comment]
I just found the answer
You know there's to lines like that:

{% load staticfiles %}
{% load static %}

at least for me I just had to erase this {% load staticfiles %} because what we can use is the second one, {% load static %}

**Comment by nadarajan-s-18673** [comment]
Oh thank you very much. You have got me out ..

**Comment by art3mis69** [comment]
https://stackoverflow.com/questions/55929472/django-templatesyntaxerror-staticfiles-is-not-a-registered-tag-library

**Comment by Awara-Amin** [comment]
Hi Guys
I like your videos a lot, thank you for sharing all of this information.
I have got a problem , please?
When I want to do search for any word and press submit button I get an error, see below for the error please. How I can sole this please?[
<img width="1440" alt="Screenshot 2020-08-27 at 01 11 54" src="https://user-images.githubusercontent.com/67450869/91369159-f539b580-e802-11ea-84d6-23dd2458681b.png">
](url)

**Comment by Awara-Amin** [comment]
I posted this question above, and I watched (Python Django Tutorial 2020 - Full Course for Beginners) I saw others had the same question! Now, you can sorted by running (python manage.py migrate) before doing the runserver. Big Big thank to you guys, I loved you videos.

**Comment by Addis-Alpha** [comment]
> I just found the answer
> You know there's to lines like that:
> 
> {% load staticfiles %}
> {% load static %}
> 
> at least for me I just had to erase this {% load staticfiles %} because what we can use is the second one, {% load static %}

thanks a lot

**Comment by kashyap3487** [comment]
> I just found the answer
> You know there's to lines like that:
> 
> {% load staticfiles %}
> {% load static %}
> 
> at least for me I just had to erase this {% load staticfiles %} because what we can use is the second one, {% load static %}

Thanks a lot
